src/openms/include/OpenMS/SYSTEM/File.h

```diff
@@ -324,6 +324,12 @@ namespace OpenMS
       @return False, if both StringLists are different (respecting the parameters)
     */
     static bool validateMatchingFileNames(const StringList& sl1, const StringList& sl2, bool basename = true, bool ignore_extension = true, bool strict = false);
+
+    /**
+      @brief Download file from given URL into a download folder. Returns when done.
```

Comment:
PLease add more info about what happens when the file exists on disk already.
What happens when you get a 404, etc.

Reply:
Throwing an exception now. Also added doc